pimpmycause/pimpmycause/templates/layout.html

```diff
@@ -43,9 +43,9 @@
                         </form>
                     {% else %}
                         <div class="navigation__login-social">
-                            <a href="#" class="navigation__social-icon">{% include "icons/linkedin.svg" %}</a>
-                            <a href="#" class="navigation__social-icon">{% include "icons/facebook.svg" %}</a>
-                            <a href="#" class="navigation__social-icon">{% include "icons/twitter.svg" %}</a>
+                            <a href="#" class="navigation__social-icon social-icon">{% include "icons/linkedin.svg" %}</a>
+                            <a href="#" class="navigation__social-icon social-icon">{% include "icons/facebook.svg" %}</a>
+                            <a href="#" class="navigation__social-icon social-icon">{% include "icons/twitter.svg" %}</a>
```

Comment:
We agreed on svgs in the past as we expect to have lots if custom assets once we get going with the dashboard, I am looking into bundling all svgs into 1 icon set instead though